docs/install-pmm/install-pmm-server/baremetal/docker/env_var.md

```diff
@@ -1,36 +1,165 @@
+    |-------------------------------------|------------------------------------|--------------------------|
+    | `PERCONA_TEST_PMM_CLICKHOUSE_ADDR`  | `PMM_CLICKHOUSE_ADDR`              |                          |
+    | `PERCONA_TEST_PMM_CLICKHOUSE_DATABASE` | `PMM_CLICKHOUSE_DATABASE`         |                        |
+    | `PERCONA_TEST_PMM_CLICKHOUSE_DATASOURCE` | `PMM_CLICKHOUSE_DATASOURCE`      |                       |
+    | `PERCONA_TEST_PMM_CLICKHOUSE_HOST`  | `PMM_CLICKHOUSE_HOST`              |                          |
+    | `PERCONA_TEST_PMM_CLICKHOUSE_PORT`  | `PMM_CLICKHOUSE_PORT`              |                          |
+    | `PERCONA_TEST_PMM_DISABLE_BUILTIN_CLICKHOUSE` | `PMM_DISABLE_BUILTIN_CLICKHOUSE` |                  |
+    | `PERCONA_TEST_PMM_CLICKHOUSE_BLOCK_SIZE` |                                    | Removed in PMM v3, new version|
+    | `PERCONA_TEST_PMM_CLICKHOUSE_POOL_SIZE`  |                                    | Removed in PMM v3, new version|
+    
+    #### PostgreSQL configuration
+    | PMM 2                               | PMM 3                              | 
+    |-------------------------------------|------------------------------------|
+    | `PERCONA_TEST_POSTGRES_ADDR`        | `PMM_POSTGRES_ADDR`                |
+    | `PERCONA_TEST_POSTGRES_DBNAME`      | `PMM_POSTGRES_DBNAME`              |
+    | `PERCONA_TEST_POSTGRES_USERNAME`    | `PMM_POSTGRES_USERNAME`            | 
+    | `PERCONA_TEST_POSTGRES_DBPASSWORD`  | `PMM_POSTGRES_DBPASSWORD`          |  
+    | `PERCONA_TEST_POSTGRES_SSL_CA_PATH` | `PMM_POSTGRES_SSL_CA_PATH`         | 
+    | `PERCONA_TEST_POSTGRES_SSL_CERT_PATH` | `PMM_POSTGRES_SSL_CERT_PATH`      | 
+    | `PERCONA_TEST_POSTGRES_SSL_KEY_PATH` | `PMM_POSTGRES_SSL_KEY_PATH`       |   
+    | `PERCONA_TEST_POSTGRES_SSL_MODE`    | `PMM_POSTGRES_SSL_MODE`            |  
+    | `PERCONA_TEST_PMM_DISABLE_BUILTIN_POSTGRES` | `PMM_DISABLE_BUILTIN_POSTGRES` |   
+   
+    #### Telemetry & development
+    | PMM 2                               | PMM 3                              | 
+    |-------------------------------------|------------------------------------|
+    | `PMM_TEST_TELEMETRY_DISABLE_SEND`   | `PMM_DEV_TELEMETRY_DISABLE_SEND`   |                
+    | `PERCONA_TEST_TELEMETRY_DISABLE_START_DELAY` | `PMM_DEV_TELEMETRY_DISABLE_START_DELAY` | 
+    | `PMM_TEST_TELEMETRY_FILE`           | `PMM_DEV_TELEMETRY_FILE`           |   
+    | `PERCONA_TEST_TELEMETRY_HOST`       | `PMM_DEV_TELEMETRY_HOST`           |   
+    | `PERCONA_TEST_TELEMETRY_INTERVAL`   | `PMM_DEV_TELEMETRY_INTERVAL`       |      
+    | `PERCONA_TEST_TELEMETRY_RETRY_BACKOFF` | `PMM_DEV_TELEMETRY_RETRY_BACKOFF` |   
+    | `PERCONA_TEST_VERSION_SERVICE_URL`  | `PMM_DEV_VERSION_SERVICE_URL`      |         
+    | `PERCONA_TEST_STARLARK_ALLOW_RECURSION` | `PMM_DEV_ADVISOR_STARLARK_ALLOW_RECURSION` |       
```

Comment:
let's drop PMM_DEV_* env variables as they are for developers.

Reply:
@idoqo, does that mean that I should delete the whole column?
<img width="768" alt="image" src="https://github.com/user-attachments/assets/0933d0ee-be0f-49e2-b7fb-3aa866a3829f">

Reply:
ok, done. thanks